### ISSUE TITLE
Improve CONSTRUCT detection

### ISSUE THREAD

**Issue by abrisse** [ruby-rdf/rdf-sesame#15]
See https://github.com/ruby-rdf/rdf-sesame/blob/develop/lib/rdf/sesame/repository.rb#L311

Today the detection of SPARQL Construct to define the default format to text/plain is quite ugly (based on regular expression /bconstruct/b). It should be removed or enhance.

**Comment by Richard-Degenne** [comment]
FWIW, according to [the SPARQL specs](https://www.w3.org/TR/sparql11-query/#sparqlGrammar), the formal definition of a Construct query is that the first word after the prologue is `CONSTRUCT`.

That shouldn't be that hard to parse properly?
